string promp_name
string path_to_data
int8 sensor_dofs #Number of individual numbers in the sensor input
int8 robot_dofs #Number of controllable robot joints
